contrib/google.golang.org/grpc/server.go

```diff
@@ -34,6 +35,9 @@ func (ss *serverStream) RecvMsg(m interface{}) (err error) {
 			ss.cfg.serverServiceName(),
 			ss.cfg.analyticsRate,
 		)
+
+		span.SetTag(ext.SpanKind, ext.SpanKindServer)
```

Comment:
Same notes here, I think we only want one of those spans marked as kind:server, and it should be the first span capture on the server side.